src/config_files.rs

```diff
@@ -125,3 +150,102 @@ pub(crate) fn create_history_path() -> Option<PathBuf> {
         }
     })
 }
+
+fn default_config_contents() -> &'static str {
+    r#"
```

Comment:
I'm wondering if we might want to use `include_str!()` so we can have a `default_config.nu` file in the repo that's a little easier to edit as changes are made.